dashboard/app/helpers/levels_helper.rb

```diff
@@ -237,7 +237,7 @@ def question_options
     level_options[:lastAttempt] = @last_attempt
     level_options.merge! @level.properties.camelize_keys
 
-    if current_user.nil? || current_user.teachers.empty?
+    if !@script.try(:professional_learning_course) && (current_user.nil? || current_user.teachers.empty?)
       # only students with teachers should be able to submit
```

Comment:
I can actually remove this, since we expect all PLC teachers to be in sections (i.e. have teachers).  I'll add a `section_required 'true'` .script attribute that mirrors `login_required 'true'`.